lighthouse-core/audits/performance-budget.js

```diff
@@ -139,7 +137,7 @@ class ResourceBudget extends Audit {
       {key: 'requestCount', itemType: 'numeric', text: str_(i18n.UIStrings.columnRequests)},
       {key: 'size', itemType: 'bytes', text: str_(i18n.UIStrings.columnTransferSize)},
       {key: 'countOverBudget', itemType: 'text', text: ''},
-      {key: 'sizeOverBudget', itemType: 'bytes', text: str_(UIStrings.columnOverBudget)},
+      {key: 'sizeOverBudget', itemType: 'bytes', text: str_(i18n.UIStrings.columnOverBudget)},
```

Comment:
@exterkamp should we be keeping in mind any particular timeline for tc roundrip of strings that we need to release in time? I guess we're only targeting an alpha for CDS but having all the metrics disappear for example would be bummer